.storybook/test-storybook-canvas.config.js

```diff
@@ -18,6 +18,7 @@ const configuration = {
             'with World viewport': DEFAULT_SCENARIO_CONFIG,
             'with Include all data viewport': DEFAULT_SCENARIO_CONFIG,
             'with disabled interactive and zoom control button': DEFAULT_SCENARIO_CONFIG,
+            'with location, size, color with alias and segmentBy with alias': DEFAULT_SCENARIO_CONFIG,
```

Comment:
I suggest `with color and segment alias shown in tooltip`

Reply:
fixed